rpi-clone

```diff
@@ -997,6 +995,25 @@ then
 	exit 0
 fi
 
+export clone=$(mktemp --tmpdir --directory rpi-clone-dst.XXXXXX)
+export clone_src=$(mktemp --tmpdir --directory rpi-clone-src.XXXXXX)
+
+if ! [ -n "$clone" -a -d "$clone" -a -n "$clone_src" -a -d "$clone_src" ]
```

Comment:
Negated tests are hard to read. I suggest to use the following test instead
```
if  [[ -z "$clone" ||  ! -d "$clone" || -z "$clone_src" || ! -d "$clone_src" ]]
```

Reply:
Totally agreed, also using `[[` as you suggest makes it even more readable by allowing `||` instead of `-o`.